style.css

```diff
@@ -99,6 +99,60 @@ h1 {
     padding: 10px;
 }
 
+.footer{
+    background-color: #F1C912;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: nowrap;
+    padding: 10vh 5vw;
+}
+.some-icons{
```

Comment:
Some inconsistency between having a space before the curly brace, and not having one. Not a biggie, but good for consistency :)